src/Data/Patch.hs

```diff
@@ -14,17 +14,20 @@ module Data.Patch
   ) where
 
 import Data.Semigroup.Commutative
-import Control.Applicative (liftA2)
 import Data.Functor.Const (Const (..))
 import Data.Functor.Identity
 import Data.Map.Monoidal (MonoidalMap)
 import Data.Proxy
+import GHC.Generics
+
+#if !MIN_VERSION_base(4,18,0)
+import Control.Applicative (liftA2)
+#endif
+
 #if !MIN_VERSION_base(4,11,0)
 import Data.Semigroup (Semigroup (..))
 #endif
-import GHC.Generics
 
-import qualified Data.Semigroup.Additive as X
```

Comment:
Oh oops, this line is a mistake because it changes the reexported interface

Reply:
@ymeister was that an accident or did you have some reasoning for that? (just making sure)